doc/installation.rst

```diff
@@ -8,36 +8,95 @@
+Before installing PyPSA-Earth on your computer, it's crucial to ensure that your system meets the necessary hardware and software requirements. The following sections outline the prerequisites in terms of hardware and software. Additionally, detailed installation guidelines for required software tools will be provided, followed by step-by-step instructions for installing PyPSA-Earth.
 
+Hardware Requirements
+=====================
+Ensure that your system meets the minimum hardware specifications to run PyPSA-Earth effectively. Recommended hardware specifications may include:
+
+* 8-16 GB RAM and adequate CPU (at-least 2-cores)
+
+* Storage (HDD/SSD) capacity depends on the region of interest. Africa model requires 40 Gb, the world `--`` 250 Gb, a single country `--` between 1-10 Gb. Tutorial requires just below 10Gb. Thus, considering all required software tools, at least 40 GB of storage space is recommended.
+
+.. note::
+
+    The subsequently described installation steps are demonstrated as shell commands, where the path before the ``%`` sign denotes the directory in which the commands following the ``%`` should be entered.
+
+Software Prerequisites
+======================
+Prior to installing PyPSA-Earth, you'll need to ensure the following software tools are installed and configured on your system:
+
+* `Miniconda <https://docs.conda.io/projects/miniconda/en/latest/miniconda-install.html>`_
+* `Git <https://git-scm.com/downloads>`_
+* `VSCode <https://code.visualstudio.com/>`_ (or any other IDE)
+* `Java <https://www.oracle.com/java/technologies/downloads/>`_
+
+Miniconda
+---------
+To use packages in python, it is highly recommended to use a ``conda`` package manager, such as `miniconda <https://docs.conda.io/projects/miniconda/en/latest/>`__. You may check if ``conda`` is already installed on your system with
+
+.. code:: bash
+
+    conda --version
+
+If ``conda`` is not installed, follow `miniconda installation guide <https://docs.conda.io/projects/conda/en/latest/user-guide/install/>`_.
+For more on information on how to install conda and work with it you can look into :ref:`software_hints`.
+
+Git
+---
+`Git <https://git-scm.com/>`__ is a free open-source tool that facilitates tracking changes in the code development and enable to coordinate the parallel software development between many developers.
+Download and install ``git`` to your system using the following `link <https://git-scm.com/downloads>`__.
+It is highly recommended to `learn the git basics <https://git-scm.com/doc>`__.
+
+VSCode
+------
+In order to write and debug python code, you need an Integrated Development Environment (IDE) that is a software used to write code. We recommend `Visual Studio Code <https://code.visualstudio.com/>`_, which is freely available online and provides an easy to use interface with Git. Obviously, any alternatives like `PyCharm <https://www.jetbrains.com/pycharm/>`_ or `Sublime <https://www.sublimetext.com/>`_ will work as well.
```

Comment:
I wonder, if it wouldn't make sense to use hyphens in `easy to use` -> `easy-to-use`?

Reply:
Yes, I agree, fixed it.